### PR TITLE
skipper: fix handling of empty value in listFlag

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
#0

### CODE REVIEW

**Comment by MustafaSaber** [comment]
👍

File: config/listflag.go [hunk 1]
```diff
@@ -34,8 +34,13 @@ func (lf *listFlag) Set(value string) error {
 		return nil
 	}
 
-	lf.value = value
-	lf.values = strings.Split(value, lf.sep)
+	if value == "" {
```
Comment (szuecs) [inline config/listflag.go:37]: this case is all defaults, and you should better test for only the `!=` case and put the code from the else branch here.

Comment (AlexanderYastrebov) [reply config/listflag.go:37]: Do you suggest to just reverse the condition or omit else branch?
It should reset previous value so we'll have else branch anyway.

Comment (szuecs) [reply config/listflag.go:37]: If it needs to reset, it's fine like that

**Comment by szuecs** [comment]
:+1:

**Pull request merged**